src/clusterfuzz/_internal/base/tasks.py

```diff
@@ -412,6 +457,36 @@ def lease(self, _event=None):  # pylint: disable=arguments-differ
     track_task_end()
 
 
+class UTaskMainPubSubTask(PubSubTask):
```

Comment:
Nit: Since `utask` in snake_case generally has no underscore, for consistency it should be `Utask` in PascalCase.

```suggestion
class UtaskMainPubSubTask(PubSubTask):
```